### PR TITLE
enhancement: Enhanced UI of the empty saved section.

### PR DESCRIPTION
**Pull request by yashk2000**
Changes: When the saved section is empty, i.e. no badges are saved, a message along with an emoji is displayed.

Screenshots for the change:

Screenshot_20190511-142206

### CODE REVIEW

**Comment by yashk2000** [comment]
@adityastic please review. And thanks for providing the emoji :)

File: app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_saved/MainSavedFragment.kt [hunk 1]
```diff
@@ -59,6 +59,20 @@ class MainSavedFragment : BaseFragment() {
         return rootView
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        if (viewModel.getFiles().value.isNullOrEmpty()) {
+            saved_text.visibility = View.GONE
```
Comment (adityastic) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_saved/MainSavedFragment.kt]: Use rootview.saved_text

**Comment by adityastic** [comment]
We should have a saved badges title though, currently only the emoji is visible. Include the title as well which stated 'saved badges'

**Comment by yashk2000** [comment]
> We should have a saved badges title though, currently only the emoji is visible. Include the title as well which stated 'saved badges'

I thought of making it invisible too when there are no badges. I thought it would look better. Tell me if it's fine. If it's not I'll not make it invisible.

File: app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_saved/MainSavedFragment.kt (hunk 1 above)
Comment (yashk2000) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/fragments/main_saved/MainSavedFragment.kt]: on it.

**Comment by adityastic** [comment]
@iamareebjamal @mariobehling @CloudyPadmal suggestions plz

**Comment by mariobehling** [comment]
Thank you. Good step forward.

Currently saved badges cannot be edited again. Please help to implement this: https://github.com/fossasia/badge-magic-android/issues/261

**Pull request merged**